Druid/DRInterpreterToCompiler.class.st

```diff
@@ -24,10 +24,11 @@ Class {
 	#superclass : #DRAbstractCompilerBuilder,
 	#instVars : [
 		'memory',
-		'compilerCompilerClass',
-		'compilerCompiler',
 		'primitiveSelector',
-		'failFirst'
+		'failFirst',
+		'primitiveCompiler',
+		'bytecodeCompiler',
+		'bytecodeSelector'
```

Comment:
Why do we have the primitiveSelector and the bytecodeSelector as instance variables? We are going to compile many primitives and bytecodes!